java/src/jmri/jmrit/logix/PortalManager.java

```diff
@@ -95,9 +95,9 @@ public Portal createNewPortal(String sName, String userName) {
         return portal;
     }
 
-    @SuppressFBWarnings(value=" ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD", justification="Lesser of two evils")
-    // Multiple instances prevented by use of the InstanceManager.  However, should multiple 
+    // Multiple instances are prevented by use of the InstanceManager.  However, should multiple 
     // instances happen a static '_nextSName' provides a chance the session can survive.
+    @SuppressFBWarnings(value=" ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD", justification="Lesser of two evils")
```

Comment:
Instead of suppressing the warning, why not make the method static?

Reply:
That would require making getBySystemName() static.  Which can't be done since it calls a method of a non-static member, _tsys.

Reply:
This seems like the wrong way to solve a problem to me.  The numbers in each instance of a manager are supposed to be unique in any case.  That's true of every other manager:  We can have "turnout 2" in separate managers, and do so all the time.  What's the "multiple managers" problem this is trying to solve?